pkg/asset/tls/etcdmetricsproxy.go

```diff
@@ -0,0 +1,177 @@
+package tls
+
+import (
+	"crypto/x509"
+	"crypto/x509/pkix"
+
+	"github.com/openshift/installer/pkg/asset"
+)
+
+// EtcdMetricsProxyCA is the asset that generates the etcd-metrics-proxy-ca key/cert pair.
+// [DEPRECATED]
```

Comment:
a new asset is already deprecated? ;)

Reply:
yeah started with your template but I see where you are going now will clean that up, thanks for the feedback!